omero/sysadmins/version-requirements.txt

```diff
@@ -777,76 +529,70 @@ Python
       - Unsupported
       - `Reference <http://legacy.python.org/dev/peps/pep-0429/>`__
     * - 3.5
-      - TBA
+      - from Sept 2015
       - TBA
       - Unsupported
       - Unsupported
       - Unsupported
       - `Reference <https://www.python.org/dev/peps/pep-0478>`__
+    * - 3.6
+      - from Dec 2016
+      - TBA
+      - Unsupported
+      - Unsupported
+      - Unsupported
+      - `Reference <https://www.python.org/dev/peps/pep-0494/>`__
 
-Distribution support
---------------------
+Version provided by distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. list-table::
     :header-rows: 1
 
     * - Version
       - CentOS/RHEL
-      - Fedora
       - Ubuntu
-      - Debian
       - Homebrew
       - FreeBSD Ports
     * - 2.6
-      - 6.x
-      - N/A
+      - 6
       - 10.04
-      - 6.0, 7.0
       - N/A
       - Yes
     * - 2.7
-      - 7.x
-      - 19, 20, 21
-      - 12.04, 13.04, 13.10, 14.04, 14.10, (15.04)
-      - 7.0, (8.0)
+      - 7
+      - 14.04, 16.04
       - Yes
       - Yes
     * - 3.2
       - N/A
       - N/A
-      - 12.04
-      - 7.0
       - N/A
       - Yes
     * - 3.3
       - N/A
-      - 19, 20
-      - 13.04, 13.10
       - N/A
       - N/A
       - Yes
     * - 3.4
       - N/A
-      - 21
-      - 14.04, 14.10, (15.04)
-      - (8.0)
-      - Yes
+      - 14.04
+      - N/A
       - Yes
     * - 3.5
       - N/A
+      - 16.04
       - N/A
+      - Yes
+    * - 3.6
       - N/A
       - N/A
-      - N/A
-      - N/A
+      - Yes
+      - Yes
     * - Details
       - 
-      - `Python 2 <https://apps.fedoraproject.org/packages/python>`__
-        `Python 3 <https://apps.fedoraproject.org/packages/python3>`__
       - `Python 2 <http://packages.ubuntu.com/search?keywords=python2&searchon=names&suite=all&section=all>`__
         `Python 3 <http://packages.ubuntu.com/search?keywords=python3&searchon=names&suite=all&section=all>`__
-      - `Python 2 <https://packages.debian.org/search?keywords=python2&searchon=sourcenames&suite=all&section=all>`__
-        `Python 3 <https://packages.debian.org/search?keywords=python3&searchon=sourcenames&suite=all&section=all>`__
       - 
```

Comment:
Text below needs the sentence "Ice 3.5 in particular has dropped 2.7 support
 for Windows, and has a significant cost in providing custom rebuilds;
 this has been rectified in Ice 3.6 with the Python module being
 installable with ``pip`` using any Python version." removing I assume - as we've dropped support for the server on Windows, this isn't relevant any more is it?